src/org/zaproxy/zap/extension/pscanrulesAlpha/ServerHeaderInfoLeakScanner.java

```diff
@@ -99,19 +98,25 @@ public int getPluginId() {
 	
 	@Override
 	public String getName(){
-		return Constant.messages.getString(MESSAGE_PREFIX + "name");
+		return Constant.messages.getString("pscanalpha.serverheader.scanner.name");

```

Comment:
Ok perfect, I was going purely from memory and thought I'd recalled something being done by name strings, glad I was wrong ;)